config/xsl/common/sections/footer.xsl

```diff
@@ -2,9 +2,21 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
 
 <xsl:template name="footer">
-    <footer>
-        Copyright (c) 2005-2023 <a href="http://www.xceptance.com/" title="Visit the Xceptance website">Xceptance Software Technologies GmbH</a>
-    </footer>
+	<xsl:param name="productUrl" select="/testreport/configuration/version/productURL" />
+    <xsl:param name="productName" select="/testreport/configuration/version/productName" />
+    <xsl:param name="productVersion" select="/testreport/configuration/version/version" />
+    <div id="footer">
+        Copyright (c) 2005-2023 <a href="http://www.xceptance.com/" title="Visit the Xceptance website">Xceptance Software Technologies GmbH</a><br />
+        Created with
+                <a href="{$productUrl}?piwik_campaign=TestReport">
```

Comment:
In the header, we use `<a href="{$productUrl}?source=TestReport">` as the URL. Use that one here as well.

Reply:
Done.